tool/tsh/kube.go

```diff
@@ -226,10 +225,12 @@ func (c *kubeLoginCommand) run(cf *CLIConf) error {
 	profile, _, err := client.Status("", cf.Proxy)
 	if err != nil {
 		if trace.IsNotFound(err) {
-			fmt.Println("Not logged in.")
-			return nil
+			return trace.Errorf("not logged in")
 		}
-		utils.FatalError(err)
+		return trace.Wrap(err)
+	}
+	if profile == nil {
```

Comment:
The comment on `client.Status` should also mention this as a possible result imo.